extensions/wikia/Email/styles/main.css

```diff
@@ -1,6 +1,81 @@
-.content {
-	padding: 15px;
+/**
+ * This is the css file for the main.mustache template
+ * Note that it's a css file, not scss, so nesting and variables are not supported
+ * Only include styles that directly affect elements inside main.mustache
+ */
+
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.body-wrap {
+	width: 100%;
+}
+
+.main-wrapper {
+	background-color: #E6EBF2;
+}
+
+.main-content {
+	background-color: #FFF;
```

Comment:
Again, you'll hate me for that, but do we have a rule for upper / lower-casing all the hex values for colors? I can see they're a bit mixed here :walking: